app/plugin_hooks_test.go

```diff
@@ -868,11 +867,10 @@ func TestUserHasBeenCreated(t *testing.T) {
 	defer tearDown()
 
 	user := &model.User{
-		Email:       model.NewId() + "success+test@example.com",
-		Nickname:    "Darth Vader",
-		Username:    "vader" + model.NewId(),
-		Password:    "passwd1",
-		AuthService: "",
+		Email:    "success+test@example.com",
+		Nickname: "testnickname",
+		Username: "testusername",
+		Password: "testpassword",
```

Comment:
About the test data, I was mainly saying that we shouldn't further copy it into new tests. This code is not part of this PR's scope so it shouldn't necessarily be edited here. In this sort of situation, it's the commit history of the individual lines that I would be worried about, but the history of these specific lines are not important, so there's not really a problem here. Thanks for cleaning this up :+1: